students/Sally_Shen/lesson2/series .py

```diff
@@ -0,0 +1,30 @@
+'''''Goal: The Fibonacci Series is a numeric series starting with the integers 0 and 1.
+
+In this series, the next integer is determined by summing the previous two'''
+
+
+def fibonacci(n):
+    if n < 2:
+        return n
+
+    a,b = 0,1
+    for _ in range(n):
+        a,b = b, a+b
+    return a
+
+print(fibonacci(4))
+
+
+def lucas(n):
+
+    a,b = 2,1
+    for _ in range (n):
+        a,b = b, a+b
+    return a
+
+print(lucas(0))
+
```

Comment:
looks like you aren't done here -- will lucas return the number for n=0 and n=1

add some more asserts to test the results!

And you still need the sum-series function...

See the template linked in the exercise description.